pkg/services/searchV2/bluge.go

```diff
@@ -435,22 +435,10 @@ func doSearchQuery(
 			fullQuery.AddShould(bluge.NewMatchAllQuery())
 		}
 	} else {
-		// The actual se
-		bq := bluge.NewBooleanQuery().
-			AddShould(bluge.NewMatchQuery(q.Query).SetField(documentFieldName).SetBoost(6)).
-			AddShould(bluge.NewMatchQuery(q.Query).SetField(documentFieldDescription).SetBoost(3)).
-			AddShould(bluge.NewMatchQuery(q.Query).
```

Comment:
ugg -- I am torn here.   This is a pretty big change that gets rid of everything nice about bluge :(

Will this behave as expected if we put both the substring and ngram query in should clauses?

Since we are not using `documentFieldDescription` we should probably remove it from the index also

Reply:
Yup :( It will behave correctly, I just removed it all for the (tiny) perf increase and consistency. Brought it back, and removed the description field from the index